src/tests/Equinor.ProCoSys.IPO.Domain.Tests/AggregateModels/PersonAggregate/SavedFilterTests.cs

```diff
@@ -29,21 +29,13 @@ public void Setup()
         public void Constructor_SetsProperties()
```

Comment:
Add a second Constructor_SetsProperties() test where you test with null for Project and Assert.IsFalse(dut.ProjectId.HasValue)

Reply:
Done